addons/payment/models/res_company.py

```diff
@@ -27,24 +27,29 @@ def _run_payment_onboarding_step(self, menu_id=None):
         """
         self.env.company.get_chart_of_accounts_or_fail()
 
-        self._install_modules(['payment_stripe'])
+        provider_code = 'razorpay' if self.currency_id.name == 'INR' else 'stripe'
```

Comment:
To make it more robost I would add an argument or something and pass it from provider instead checking code each time `provider_code` :thinking: